src/benchmark/Benchmark.zig

```diff
@@ -131,6 +131,13 @@ pub const VTable = struct {
 };
 
 test Benchmark {
+    // This test fails on FreeBSD so skip:
+    //
+    // /home/runner/work/ghostty/ghostty/src/benchmark/Benchmark.zig:165:5: 0x3cd2de1 in decltest.Benchmark (ghostty-test)
+    //     try testing.expect(result.duration > 0);
```

Comment:
This isn't unique to FreeBSD, this happens because this test is timing dependent, which is naughty. We should fix that (not in this PR).